libimage/load.go

```diff
@@ -142,7 +152,21 @@ func (r *Runtime) loadMultiImageDockerArchive(ctx context.Context, ref types.Ima
 	// should.
 	path := ref.StringWithinTransport()
 	if err := fileutils.Exists(path); err != nil {
-		return r.copyFromDockerArchive(ctx, ref, options)
+		images, err := r.copyFromDockerArchive(ctx, ref, options)
+		if err != nil {
+			return nil, err
+		}
+		names := []string{}
+		for _, img := range images {
+			if len(img.Names()) > 0 {
+				names = append(names, img.Names()...)
```

Comment:
All of the names, incl. names of all the deduplicated images which were not present in the archive?